Dockerfile.image-service-mce

```diff
@@ -32,6 +84,7 @@ ENV DATA_DIR=$DATA_DIR
 
 RUN dnf -y update && dnf install -y cpio squashfs-tools && dnf clean all
 
+COPY --from=tools /usr/local /usr/local
```

Comment:
Should/Can we just copy what we need to ensure we're not overwriting anything here?

Reply:
Yes, modified this to copy the very minimum that we need: the `dump.erofs` binary and the `liblzma.so.5` shared library.